components/public-api-server/pkg/apiv1/team.go

```diff
@@ -126,6 +130,8 @@ func teamMembersToAPIResponse(members []*protocol.TeamMemberInfo) []*v1.TeamMemb
 			UserId:      m.UserId,
 			Role:        teamRoleToAPIResponse(m.Role),
 			MemberSince: parseTimeStamp(m.MemberSince),
+			AvatarUrl:   m.AvatarUrl,
```

Comment:
This is the main change. The rest is test cleanup to reduce duplication and make it easier to define new test structs.